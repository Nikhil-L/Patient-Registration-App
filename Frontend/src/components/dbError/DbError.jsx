import React from "react";

/**
 * This component renders the error message on to the screen
 * whenever the server is out of service.
 */
export class DbError extends React.Component{

    /**
     * This returns the html content
     * @returns Html for displaying error message
     */
     render()
     {
         return(<h2>Oops! Server seems to be down try again after some time</h2>);
     }
}