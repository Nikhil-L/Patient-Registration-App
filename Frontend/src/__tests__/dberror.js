import React from "react";
import '@testing-library/jest-dom'
import {DbError} from "../components/dbError/DbError"
import renderer from "react-test-renderer"


/**
 * This is to test the DbError component
 */
describe("Details test", ()=>{

  /**
   * Snapshot testing
   */
  it("matches a snapshot", ()=>{
    const tree = renderer.create(<DbError/>).toJSON();
    expect(tree).toMatchSnapshot();
  })

})