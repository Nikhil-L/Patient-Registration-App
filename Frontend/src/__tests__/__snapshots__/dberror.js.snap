// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details test matches a snapshot 1`] = `
<h2>
  Oops! Server seems to be down try again after some time
</h2>
`;
